games/o.yaml

```diff
@@ -1313,6 +1313,38 @@
   video:
     youtube: 5CXqWdbWrVs
 
+- name: OpenKKnD
```

Comment:
please add images and/or a video

Reply:
It seems like OpenKKnD is a de-compilation project. No screenshots or videos found.
Different story for for OpenKrush however. I've added some.